photon_installer/ks_config.txt

```diff
@@ -410,4 +410,25 @@ Kickstart config file is a json format with following possible parameters:
         Default value: "photon:latest"
 	Example: { "photon_docker_image": "photon:3.0" }
 
+"repos" (optional)
+	Specify one or more RPM repos to install the rpms from
+	Default value: [
```

Comment:
Please add feature key as well in Default value and in Example.

"repos" : [ {} ]